subsys/bootloader/image/CMakeLists.txt

```diff
@@ -90,6 +93,17 @@ add_custom_target(
   signature_public_key_file_target
   )
 
+if (CONFIG_SOC_NRF9160)
```

Comment:
Have you tried using objcopy? As we do in the `mcuboot/zephyr/CMakeLists.txt` to move the hex file?

Reply:
Wasn't aware, thanks. I ended up avoiding moving the hex altoghether.